flink-streaming-java/src/main/java/org/apache/flink/streaming/api/windowing/assigners/SlidingAlignedProcessingTimeWindows.java

```diff
@@ -35,7 +35,7 @@
  * <p>
  * <b>WARNING:</b> Bear in mind that no rescaling and no backwards compatibility is supported.
  * */
-public class SlidingAlignedProcessingTimeWindows extends BaseAlignedWindowAssigner {
+public final class SlidingAlignedProcessingTimeWindows extends BaseAlignedWindowAssigner {
```

Comment:
This class does not have an annotation.
Should we add `@PublicEvolving`?

Same is true for the `TumblingAlignedProcessingTImeWindows`.